index.html

```diff
@@ -106,49 +131,58 @@ <h2 class="projects-title">Multi-Post<br>Stories</h2>
                 <li class="project-list">javaScript</li>
              </ul>
              <button class="projects-btn">See Project</button>
+            </div>   
         </div>
     </section>
 
     <section class="about-myself">
-        <h2 class="about-me-headline">About<br>Myself</h2>
+        <div class="aboutme-desk-left">
+            <h2 class="about-me-headline">About<br>Myself</h2>
         <p class="about-me-description">Hello I’m a software developer! 
             I can help you build a product ,
             feature or website Look through some of my work and experience!
             If you like what you see and have a project you need coded, 
             don’t hestiate to contact me.</p>
             <p class="social2">LET'S CONNECT</p>
             <ul class="social-icons2">
-                <li><img src="images/twitter.png" alt=""></li>
-                <li><img src="images/linkedin.png" alt=""></li>
-                <li><img src="images/M.png" alt=""></li>
-                <li><img src="images/github.png" alt=""></li>
-                <li><img src="images/hand.png" alt=""></li>
+                <li><a href=""><img src="images/twitter.png" alt=""></a></li>
+                <li><a href=""><img src="images/linkedin.png" alt=""></a></li>
+                <li><a href=""><img src="images/M.png" alt=""></a></li>
+                <li><a href=""><img src="images/github.png" alt=""></a></li>
+                <li><a href=""><img src="images/hand.png" alt=""></a></li>
```

Comment:
- [ ] Please make sure that your design is the same as the figma template as much as possible. kindly follow spacing(margin, padding) in your desktop version. 
### Your design👇
![aboutst](https://user-images.githubusercontent.com/66992901/199991534-163444ab-3454-42a5-8962-63367cd2b3e9.PNG)

### Figma template👇
![aboutfigma](https://user-images.githubusercontent.com/66992901/199990893-a273e93c-ffdf-43e6-8a0f-5936e5ef6d87.PNG)

Reply:
Done

Reply:
@Felix45 Hello, do you mean the margins are not matched? please can you specify where is the exact issue